systemtests/tests/notls/testrunner

```diff
@@ -52,7 +51,7 @@ messages
 @#
 @$out $tmp/log2.out
 wait
-restore client=bareos-fd fileset=SelfTest where=$tmp/bareos-restores select all done
+restore client=bareos-fd where=$tmp/bareos-restores select all done
```

Comment:
was changing these intentional?

Reply:
Yes. It is not required to specify the fileset on restore.